extension-files/consent-creator.js

```diff
@@ -1,131 +1,201 @@
-<<<<<<< HEAD
-/*
-/*
-Information on API used can be found at Mozilla WebExtensions documentation
-https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/cookies 
-*/
-	/*
-	NAME: initiateCookie
-	PURPOSE: processes "tabs" variable and sends it to removeCookies and addGVCC
-	*/
-	function initiateCookie()
-	{
-		function success(e)
-		{
-			browser.webNavigation.onCommitted.removeListener(initiateCookie);
-			console.log("cookie");
-		}
-		/*
-		NAME: addGVCC
-		PURPOSE: Creates a new GVCC and adds it to the browser
-		*/
-		function addGVCC(tabs)
-		{
-			//100% consent "BOSl-jdOSl-jlABABBENBd-AAAAgV______________________________________________________________________________________A"
-			//0% consent "BOSmIULOSmIUTABABBENBdAAAAAgWAAA"
-			//this seems to work pretty well for every website
-			domain = tabs[0].url.split('/')[2];
-				var cookieset = browser.cookies.set
-				({
-					url: tabs[0].url,
-					name: "euconsent",
-					value: "BOSmIULOSmIUTABABBENBdAAAAAgWAAA",
-					httpOnly: false,
-					//temporary
-					expirationDate: 1568133479,
-					path: "/",
-					firstPartyDomain: "",
-					storeId: "firefox-default",
-					domain: domain
-				})
-				cookieset.then(success);
-		}
-	//Retrieves information of current tab.
-	var getActive = browser.tabs.query
-	({
-		active: true,
-		currentWindow: true
-	});
-	getActive.then(addGVCC);
-	}
-	console.log("Begin consent-creator.js");
-	//Listeners for actions that would provoke a dialog
-	browser.tabs.onActivated.addListener(initiateCookie)
-	browser.webNavigation.onCommitted.addListener(initiateCookie);
-	browser.webNavigation.onDOMContentLoaded.addListener(initiateCookie);
-	
-=======
 "use strict";
-
-/**
+ /**
  * Processes "tabs" variable and sends it to removeCookies and addGVCC
  */
 async function initiateCookie() {
-	browser.webNavigation.onCommitted.removeListener(initiateCookie);
-	let tabs = await browser.tabs.query({active: true, currentWindow: true});
-	var urlSet = tabs[0].url;
-	var nameSet = "euconsent";
-	var valueSet = "No value set!";
-	var httpOnlySet = false;
-	var pathSet = "/";
-	var firstPartyDomainSet = "";
-	var storeIdSet = "firefox-default";
+    let tabs = await browser.tabs.query({active: true, currentWindow: true});
+    let domain = tabs[0].url.split("/")[2];
+    browser.webNavigation.onCommitted.removeListener(initiateCookie);
+    let urlSet = tabs[0].url;
+    let nameSet = "euconsent";
+    let valueSet = "BOSl-jdOSl-jlABABBENBd-AAAAgV___________" +
```

Comment:
Is this the "all-vendor consent" cookie?  If so, please add a comment saying that, and listing the websites (from the list we are interested in) where it works.

Also, doesn't setting the general "all vendor no consent" hide all the same dialogs?

Reply:
I added a comment declaring that this is the default set of values for the "euconsent" cookie with full vendor inclusion.

Not every website uses euconsent (actually very few use it to standard by what I can tell) and almost every website I've removed the banner on has required an extra setting, or a different cookie altogether. These are still cookies working for GDPR compliance, and many of them mention it in their banners.